### DISCUSSION TITLE
Grafana 8 alert notifications have to much details ... how to remove 'em ?

### DISCUSSION THREAD

**Discussion by OmarGhoche** [grafana/grafana#50789] [Alerting]
The new Grafana 8 unified alerting settings are quite harsh to get used to ... most I got ... but the notifications are now too technical and I found no way to toggle off the many text "junk" that comes along (like `alertname`, `instance`, etc). 

I just needed the description part.

![image](https://user-images.githubusercontent.com/17257630/173592982-34e5bf5e-ab83-4ec5-825b-ad8ccbfc65a0.png)

We are designing alert notifications for a not-so-technical team to keep updated with any deviance of their data showed on Grafana ... the legacy alerting system worked just fine!

Found nothing here nor online on how to toggle off the details ... I would apreciate any help.

Thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
